server-embedded/src/main/java/org/exoplatform/chat/server/ChatServer.java

```diff
@@ -113,7 +114,23 @@ public Response.Content index() throws IOException
   {
     return index.ok();
   }
-
+  
+  @Resource
+  @Route("/filterOutSilentUsers")
+  public Response.Content getFilteredList(String user,List<String> roomUsers, String roomId, String token){
+	    if (!tokenService.hasUserWithToken(user, token)){
+	      return Response.notFound("Petit malin !");
+	    }
+	  List<String> receivers =  new ArrayList<>();
+	  for(String participant : roomUsers.get(0).split(",")) {
```

Comment:
Why the variable roomUsers is not cast to an array ? why the need to get the element at index 0 ?
I think there is a change to do in the way you call it.